File: app/src/main/java/com/islam/retrofitsoap/webservice/request/simplexml/RequestBody.java
package com.islam.retrofitsoap.webservice.request.simplexml;

import org.simpleframework.xml.Element;
import org.simpleframework.xml.Root;

@Root(name = "soapenv:Body", strict = false)
public class RequestBody {

    @Element(name = "getWeatherbyCityName", required = false)
    public RequestModel getWeatherbyCityName;
}

File: app/src/main/java/com/islam/retrofitsoap/webservice/request/tikxml/RequestBody.kt
package com.islam.retrofitsoap.webservice.request.tikxml

import com.tickaroo.tikxml.annotation.Element
import com.tickaroo.tikxml.annotation.Xml

@Xml(name = "soapenv:Body")
class RequestBody {
    @Element(name = "getWeatherbyCityName")
    var getWeatherbyCityName: RequestModel? = null
}